mycurd/src/main/groovy/daos/UserDao.groovy

```diff
@@ -0,0 +1,76 @@
+package daos
+
+import com.google.inject.Inject
+import com.webonise.mycurd.Tables
+import com.webonise.mycurd.tables.pojos.Studentinfo
+import groovy.transform.CompileStatic
+import org.jooq.DSLContext
+import com.webonise.mycurd.tables.records.StudentinfoRecord;
+import org.jooq.Param
+import org.jooq.Record
+import org.jooq.SelectJoinStep
+import org.jooq.TableLike
+import org.jooq.impl.DSL
+import pojos.StudentPojo
+
+/**
+ * Created by webonise on 25-12-2016.
+ */
+
+@CompileStatic
+class UserDao {
+    private static final String ID_PARAMETER = "ID_PARAMETER"
+    private final DSLContext dslContext
+
+    @Inject
+    UserDao(DSLContext dslContext) {
+        this.dslContext = dslContext
+    }
+
+    List<StudentPojo> allStudents() {
+        //println(dslContext.select().from(Tables.STUDENTINFO).fetch().into(Tables.STUDENTINFO))
+        List<StudentinfoRecord> studentinfoRecordList = dslContext.select().from(Tables.STUDENTINFO).fetch().into(Tables.STUDENTINFO)
+        List<StudentPojo> studentPojoList=new ArrayList<StudentPojo>();
+        for (int rowCounter = 0; rowCounter < studentinfoRecordList.size(); rowCounter++) {
+           StudentPojo studentPojo = new StudentPojo(String.valueOf(studentinfoRecordList.get(rowCounter).get(1)),
+                Integer.parseInt(String.valueOf(studentinfoRecordList.get(rowCounter).get(2))),
+                    String.valueOf(studentinfoRecordList.get(rowCounter).get(3)),
+                        Integer.parseInt(String.valueOf(studentinfoRecordList.get(rowCounter).get(4))))
+            studentPojoList.add(studentPojo);
+        }
+        return studentPojoList
+    }
+    int saveStudent(StudentPojo student) {
+        println "-----"+student
```

Comment:
please dont use `println`.use the annotation `@Slf4j` on the class.it gives you a logger for free